### PR TITLE
Sort imports, add eslint based sort plugin.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

**Comment by devinleighsmith** [comment]
I realized that this will create nightmare merges for anyone working on the FE. Planning on merging this when we are closer to the cutoff for this sprint.

File: frontend/package.json [hunk 1]
```diff
@@ -101,6 +101,7 @@
     "enzyme-to-json": "3.6.2",
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-prettier": "3.4.0",
+    "eslint-plugin-simple-import-sort": "^7.0.0",
```
Comment (Fosol) [inline frontend/package.json:104]: ^7.0.0

File: frontend/src/components/Table/ColumnSort.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
+import variables from '_variables.module.scss';
```
Comment (Fosol) [inline frontend/src/components/Table/ColumnSort.tsx:1]: No change required for this PR.  This looks like something that could be replaced by the styled theme variables now.

**Review by Fosol — APPROVED** [review]
Why did all the imports get sorted?  Will they be sorted automatically going forward?  Does it require a vscode extension?

**Comment by devinleighsmith** [comment]
> Why did all the imports get sorted? Will they be sorted automatically going forward? Does it require a vscode extension?

Not sure how to answer the first question, the entire purpose of this PR is to sort the imports. Maybe the question is why is that important? The obvious answer is neatness and consistency. Less obvious is the import of external scss needs to be handled in a consistent order otherwise webpack will throw an error.

With this change imports should be sorted by eslint automatically, no extension required.

**Pull request merged**